appbuilder/services/npm-service.ts

```diff
@@ -186,8 +187,13 @@ export class NpmService implements INpmService {
 		return !!(matches && matches[0]);
 	}
 
-	public getScopedDependencyInformation(dependency: string): IScopedDependencyInformation {
-		let matches = dependency.match(NpmService.SCOPED_DEPENDENCY_REGEXP);
+	public getDependencyInformation(dependency: string): IDependencyInformation {
+		let regExp = this.isScopedDependency(dependency) ? NpmService.SCOPED_DEPENDENCY_REGEXP : NpmService.DEPENDENCY_REGEXP;
+		return this.getDependencyInformationCore(dependency, regExp);
+	}
+
+	private getDependencyInformationCore(dependency: string, regExp: RegExp): IDependencyInformation {
```

Comment:
as this method is not used anywhere else, I vote for merging it with `getDependencyInformation`